### PR TITLE
Pep8 and travis configuration

### PR DESCRIPTION
**Pull request by Muddasser**

### CODE REVIEW

**Comment by Muddasser** [comment]
@benpatterson @raeeschachar Please review.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: python
```
Comment (benpatterson) [inline .travis.yml]: please also add `sudo: false` to this file. That will cause Travis to use docker containers, which start much faster.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+python:
+  - "2.7"
+# command to install dependencies
```
Comment (benpatterson) [inline .travis.yml]: you can remove this comment.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+install:
+  - pip install -r requirements/travis_requirement.txt
```
Comment (benpatterson) [inline .travis.yml]: for now, let's just use the same requirements file...no need for a specific one for travis.

**Comment by benpatterson** [comment]
Done with my first pass.

**Comment by Muddasser** [comment]
@muhammad-ammar Please review.

**Comment by benpatterson** [comment]
👍

**Comment by raeeschachar** [comment]
👍

**Pull request merged**